test/query.js

```diff
@@ -902,14 +933,26 @@ describe('Queries', function () {
 
     it('should perform a background query that executes the operations #slow', async function () {
       const query = client.query(helper.namespace, testSet)
-      const ops = [op.incr('backgroundOps', 1)]
+      const ops = [op.write('backgroundOps', 1)]
       const job = await query.operate(ops)
       await job.waitUntilDone()
 
       const key = keys[Math.floor(Math.random() * keys.length)]
       const record = await client.get(key)
       expect(record.bins.backgroundOps).to.equal(1)
     })
+
+    it('should set TTL to the specified value #slow', async function () {
+      const query = client.query(helper.namespace, testSet)
+      query.ttl = 45
```

Comment:
i know it's just a test, but people use this for examples and 45 seconds sounds a bit harsh, maybe using 1h?

Reply:
Makes sense to me, I will add this in before releasing.